Sources/TuistKit/Commands/ScaffoldCommand.swift

```diff
@@ -126,6 +134,15 @@ extension ScaffoldCommand {
         ScaffoldCommand.optionalTemplateOptions = optional.map {
             (name: $0, option: Option<String?>())
         }
+        ScaffoldCommand.plugins = plugins
+    }
+
+    private static func loadPlugins(at path: String?) throws -> Plugins {
+        guard let path = path.map({ AbsolutePath($0, relativeTo: FileHandler.shared.currentPath) }) else { return .none }
+        let configLoader = ConfigLoader(manifestLoader: ManifestLoader())
+        let pluginService = PluginService()
+        let config = try configLoader.loadConfig(path: path)
+        return try pluginService.loadPlugins(using: config)
```

Comment:
To make this testable, it might be better to move this logic into `ScaffoldService` as we should try to keep `ScaffoldCommand` to be as business-logic-free as possible (similarly to how we extract the options from `ScaffolService.loadTemplateOptions` function)